target/start-mailserver.sh

```diff
@@ -122,6 +122,7 @@ function register_functions
 
   _register_setup_function "_setup_dkim"
   _register_setup_function "_setup_ssl"
+  _register_setup_function "_setup_postfix_aliases"
```

Comment:
What's this about?:)

Reply:
Sloppiness.  I was moving it around to test why aliases do not get created when using default mailserver.env.  It's still a mystery.